gradle/dependencies.gradle

```diff
@@ -23,6 +23,7 @@ ext.deps = [
         kotlin: "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}",
         dokka: "org.jetbrains.dokka:dokka-gradle-plugin:${versions.dokka}",
         intellij: "org.jetbrains.intellij.plugins:gradle-intellij-plugin:1.1.2",
+        grammarKit: "gradle.plugin.org.jetbrains.intellij.plugins:gradle-grammarkit-plugin:2021.1.3",
```

Comment:
im surprised we needed to add this if it wasnt there before

Reply:
It's not needed as such - just upgrades the Gradle plugin version to something newer. There have been improvements to Gradle compatibility and fixes to deprecation warnings etc. The version of Grammar-Kit dependency itself is then set with `grammarKitRelease` in `sqldelight-compiler/build.gradle`.